upstream-community-operators/postgresql/4.5.1/postgresoperator.v4.5.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,810 @@
+          - create
+          - patch
+          - update
+          - delete
+          - deletecollection
+        - apiGroups:
+          - crunchydata.com
+          resources:
+          - pgclusters
+          - pgpolicies
+          - pgreplicas
+          - pgtasks
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - patch
+          - update
+          - delete
+          - deletecollection
+      permissions:
+      - serviceAccountName: postgres-operator
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          - secrets
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - serviceaccounts
+          verbs:
+          - get
+      deployments:
+      - name: postgres-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: postgres-operator
+              vendor: crunchydata
+          template:
+            metadata:
+              labels:
+                name: postgres-operator
+                vendor: crunchydata
+            spec:
+              serviceAccountName: postgres-operator
+              containers:
+              - name: apiserver
+                image: registry.developers.crunchydata.com/crunchydata/pgo-apiserver:centos7-4.5.1
+                imagePullPolicy: IfNotPresent
+                ports:
+                - containerPort: 8443
+                readinessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8443
+                    scheme: HTTPS
+                  initialDelaySeconds: 15
+                  periodSeconds: 5
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8443
+                    scheme: HTTPS
+                  initialDelaySeconds: 15
+                  periodSeconds: 5
+                env:
+                - name: NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: PGO_INSTALLATION_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: PGO_OPERATOR_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: CRUNCHY_DEBUG
+                  value: "false"
+                - name: EVENT_ADDR
+                  value: localhost:4150
+                - name: PORT
+                  value: "8443"
+                - name: CCP_IMAGE_PREFIX
+                  value: registry.developers.crunchydata.com/crunchydata
+                - name: CCP_IMAGE_TAG
+                  value: centos7-12.4-4.5.1
```

Comment:
@jkatz this does not seem to be the good tag. Should be `centos7-12.5-4.5.1` instead of `centos7-12.4-4.5.1` as the following image `registry.developers.crunchydata.com/crunchydata/crunchy-postgres-ha:centos7-12.4-4.5.1` is not available in the registry.
Every occurence of `centos7-12.4-4.5.1`should be replaced.

Reply:
*facepalm* Sorry, silly mistake. I'll have that fixed shortly.